toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -960,6 +993,16 @@ func hydrateFromRemoteSource(fileHydrationState map[string]bool, newSourceDir st
 
 		fileHydrationState[fileName] = true
 		logger.Log.Debugf("Hydrated (%s) from (%s)", fileName, url)
+
+		if cacheMissingFiles && srcConfig.localSourceCacheDir != "" {
+			cacheDestinationFile := filepath.Join(srcConfig.localSourceCacheDir, fileName)
+			err = file.Copy(destinationFile, cacheDestinationFile)
```

Comment:
I am wondering, if we created a mount point for this, as I was doing some local testing.
I could not see the `tars` getting populated in the `CACHE_DIR`, although, they exist in the chroot `CACHE_DIR`.
Double checking here, before we merge it, do we need to modify the `func createChroot` to ensure that the cache is populated in the host box.

Something like this:
```go
func createChroot(workerTar, buildDir, outDir, specsDir, cacheDir string) (chroot *safechroot.Chroot, newBuildDir, newOutDir, newSpecsDir string, err error) {
        const (                                                                                                                                                                                                                                                                                                             
                chrootName       = "srpmpacker_chroot"                                                                                                                                                                                                                                                                      
                existingDir      = false                                                                                                                                                                                                                                                                                    
                leaveFilesOnDisk = false                                                                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                                                                                                            
                outMountPoint    = "/output"                                                                                                                                                                                                                                                                                
                specsMountPoint  = "/specs"                                                                                                                                                                                                                                                                                 
                buildDirInChroot = "/build"                                                                                                                                                                                                                                                                                 
        )                                                                                                                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                                                                                            
        extraMountPoints := []*safechroot.MountPoint{                                                                                                                                                                                                                                                                       
                safechroot.NewMountPoint(outDir, outMountPoint, "", safechroot.BindMountPointFlags, ""),                                                                                                                                                                                                                    
                safechroot.NewMountPoint(specsDir, specsMountPoint, "", safechroot.BindMountPointFlags, ""),                                                                                                                                                                                                                
                safechroot.NewMountPoint(cacheDir, cacheDir, "", safechroot.BindMountPointFlags, ""),  // this seems to be missing                                                                                                                                                                                                                      
        } 
...
}
```

Reply:
Seems you're correct- must have messed up the testing somewhere. Will fix, thanks!

Reply:
Addressed this problem by adding a cache mount folder as well as copying in/out the cache folder's contents during docker-based builds.